src/runtime/node/https/index.ts

```diff
@@ -1,26 +1,23 @@
 // https://nodejs.org/api/https.html
-import type nodeHttps from "node:https";
+import type https from "node:https";
 import { notImplemented, notImplementedClass } from "../../_internal/utils";
 import mock from "../../mock/proxy";
 
-export const Server: typeof nodeHttps.Server =
+export const Server: typeof https.Server =
```

Comment:
```suggestion
export const Server: typeof nodeHttps.Server =
```

Reply:
Ok, I wanted to make it consistent with http.
Now I know the current way ;)
Thanks.